### ISSUE TITLE
Shader Graph can't compile Foliage, Cloud2D, and Grass shaders

### ISSUE THREAD

**Issue by hunterpankey** [dustyroom-studio/quibli-doc#10]
**Describe the bug**
These shaders depend on StylizedLighting.hlsl by relative path, somewhere in the Shader Graph or shader properties. When I import packs, I move them into an Asset Packs or Tools folder under Assets, so the relative path to StylizedLighting.hlsl changes. I need to find those references and update them to the correct path.

**To Reproduce**
1. Import Quibli into a new URP project.
2. Create a separate folder under Assets, e.g., "Asset Packs".
3. Move the Quibli folder under the folder created in step 2.
4. Load the city or other grass/foliage using scene and check objects that use those shaders.
5. Objects are in "Unity Pink" because the shader couldn't be compiled because of the broken references.

**Expected behavior**
Eh, it's not entirely unexpected, but I would expect to be able to change the relative path to the dependency so it can be picked up properly. A good outcome would be if there were a setting in Shader Graph or in a configuration object in the Quibli package that could be updated, or if it looked for StylizedLighting.hlsl itself.

**Screenshots**
![image](https://user-images.githubusercontent.com/7977119/164788781-a98d8711-ec11-4c41-9aa3-ef613f7962a9.png)
![image](https://user-images.githubusercontent.com/7977119/164788827-063facff-54ea-4f28-895d-3aa6e31d41e0.png)

**Unity details**:
Quibli version 1.4.8
Unity 2020.3.33f1
Dev platform: WindowsEditor
Target platform: StandaloneWindows64
URP installed: True, version 10.8.1
Render pipeline: UniversalPipeline,LightweightPipeline
Color space: Linear
Quality config: Quibli URP Config
Graphics config: Quibli URP Config

**Comment by dustyroom-studio** [comment]
Hi! This is not us referencing the files by absolute paths, this is Shader Graph in Unity 2020 being silly.
Can you please try right-clicking on `Asset Packs/Quibli/Shaders` and selecting "Reimport"?

**Comment by hunterpankey** [comment]
Yeah, I thought there was a chance it was Shader Graph once I did a text search for the parts of the StylizedLighting.hlsl path and came up empty for the whole project. I saw that response in the Trello board but didn't have access to the whole conversation, so I wasn't sure if I had the same issue as the other reporter.

At any rate, I think I tried reimporting without success, but I'll give it another go and see if I was just going fast and didn't reimport the right thing.

Thanks for responding so quickly!

**Comment by hunterpankey** [comment]
Yes, reimporting the whole folder seems to have done it. I think I had only reimported the Foliage shader before. The bushes and trees now render as expected, though the three shaders were still listed under the Failed to Compile category. I had to open the Shader Graph window for each and save them before they were recategorized correctly. So this is basically just a documentation issue, I think. But regardless, thank you for diving in and helping me!
